pkg/actions/addon/create.go

```diff
@@ -25,9 +25,17 @@ const (
 )
 
 func (a *Manager) Create(addon *api.Addon) error {
+	version := addon.Version
+	if addon.Version == "latest" {
+		var err error
+		version, err = a.getLatestVersion(addon)
+		if err != nil {
+			return fmt.Errorf("failed to fetch latest addon version: %w", err)
+		}
+	}
 	createAddonInput := &eks.CreateAddonInput{
 		AddonName:    &addon.Name,
-		AddonVersion: &addon.Version,
+		AddonVersion: &version,
 		ClusterName:  &a.clusterConfig.Metadata.Name,
 		//ResolveConflicts: 		"enum":["OVERWRITE","NONE"]
```

Comment:
is this commented bit needed?

dejavu... have i asked this before? 🤔

Reply:
yes you have :see_no_evil: , I found it useful to remind me what the values are for `ResolveConflicts` since its an enum

Reply:
aaaaah

feel free to pin a link to this comment in our slack, in the inevitable case that i ask this again some day

Reply:
or add a comment above explaining why it is there